src/test/java/com/commercetools/sync/producttypes/utils/ProductTypesUpdateActionUtilsTest.java

```diff
@@ -0,0 +1,99 @@
+package com.commercetools.sync.producttypes.utils;
+
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.products.attributes.AttributeDefinition;
+import io.sphere.sdk.products.attributes.AttributeDefinitionBuilder;
+import io.sphere.sdk.products.attributes.AttributeDefinitionDraft;
+import io.sphere.sdk.products.attributes.AttributeDefinitionDraftBuilder;
+import io.sphere.sdk.products.attributes.StringAttributeType;
+import io.sphere.sdk.producttypes.ProductType;
+
+import static com.commercetools.sync.producttypes.utils.ProductTypeUpdateActionUtils.buildChangeDescriptionAction;
+import static com.commercetools.sync.producttypes.utils.ProductTypeUpdateActionUtils.buildChangeNameAction;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.sphere.sdk.producttypes.ProductTypeDraft;
+import io.sphere.sdk.producttypes.commands.updateactions.ChangeDescription;
+import io.sphere.sdk.producttypes.commands.updateactions.ChangeName;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+
+public class ProductTypesUpdateActionUtilsTest {
```

Comment:
don't forget to update the test class name after you changed the actual class name ;)

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/287/commits/20757da5f160e84367e6f63bca6de57cda18e99f